### ISSUE TITLE
Maven Central checksum error

### ISSUE THREAD

**Issue by mtnygard** [TomasMikula/UndoFX#5]
When fetching org.fxmisc.undo/undofx from Maven central, I get the following checksum error:

```
Could not find metadata org.fxmisc.undo:undofx/maven-metadata.xml in local (/Users/mtnygard/.m2/repository)
Could not transfer metadata org.fxmisc.undo:undofx/maven-metadata.xml from/to central (https://repo1.maven.org/maven2/): Checksum validation failed, expected 5aa23ca89ce7f15b4e877d85d2dca9b554f44b18 but is 8805fc8f574b9614ac518a4c71bc17ae08a64007
```

We are reaching this dependency via RichTextFX version 0.6.4

**Comment by bigDevDe** [comment]
Hi Tomas,
I have the same Issue. The Checksum of the sha1-xml is really

8805fc8f574b9614ac518a4c71bc17ae08a64007.

Best regards 
Michael

**Comment by TomasMikula** [comment]
Sorry for late response. I didn't encounter this problem when building projects with Gradle. Maybe Gradle does not check the checksum? What build tool do you use? Anyway, I published a new release 1.1.1 that should fix the problem. Can you confirm? Thanks!
